code/datums/components/food/edible.dm

```diff
@@ -505,6 +505,8 @@ Behavior that's still missing from this component that original food items had t
 			food_taste_reaction = FOOD_DISLIKED
 		else if(foodtypes & H.dna.species.liked_food)
 			food_taste_reaction = FOOD_LIKED
+	if(food_flags & FOOD_SILVER_SPAWNED) // it's not real food
+		food_taste_reaction = FOOD_TOXIC
```

Comment:
What does this have to do with a table manners pr

Reply:
in a PR to make people actually go eat food from the chef, leaving garbage like xenobio food in is stupid, if you want to eat xenobio food take the trait that makes you ignore food toxicity

Reply:
I don't see what food availability has to do with what you have to do to successfully eat the food